src/test/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/Record2SolutionMappingTranslationTest.java

```diff
@@ -0,0 +1,332 @@
+import se.liu.ida.hefquin.engine.data.SolutionMapping;
+import se.liu.ida.hefquin.engine.data.impl.SolutionMappingImpl;
+import se.liu.ida.hefquin.engine.federation.access.Neo4JException;
+import se.liu.ida.hefquin.engine.query.impl.TriplePatternImpl;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.data.TableRecord;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.data.impl.LPGNode;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.impl.DefaultConfiguration;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.impl.Record2SolutionMappingTranslatorImpl;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.impl.SPARQLStar2CypherTranslatorImpl;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.query.CypherQuery;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.utils.CypherUtils;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.utils.CypherVarGenerator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class Record2SolutionMappingTranslationTest {
+
+    final LPGNode node1 = new LPGNode("1", null, null);
+    final LPGNode node2 = new LPGNode("2", null, null);
+    final LPGNode node3 = new LPGNode("3", null, null);
+    final LPGNode node4 = new LPGNode("4", null, null);
+    final LPGNode node5 = new LPGNode("5", null, null);
+    final LPGNode node6 = new LPGNode("6", null, null);
+    final LPGNode node9 = new LPGNode("9", null, null);
+    final LPGNode node14 = new LPGNode("14", null, null);
+    final LPGNode node22 = new LPGNode("22", null, null);
+
+    @Test
+    public void translateVarPropertyLiteralTest() throws JsonProcessingException, Neo4JException {
+        final LPG2RDFConfiguration conf = new DefaultConfiguration();
+        final Triple t = new Triple(Var.alloc("s"), conf.mapProperty("name"),
+                NodeFactory.createLiteral("Lana Wachowski"));
+        final SPARQLStar2CypherTranslator translator = new SPARQLStar2CypherTranslatorImpl();
+        final CypherQuery translation = translator.translateTriplePattern(new TriplePatternImpl(t), conf);
+        final CypherVarGenerator generator = translator.getVarGenerator();
+        final String neo4jResponse = "{\"results\":[{\"columns\":[\"ret1\"],\"data\":[" +
+                "{\"row\":[{\"born\":1965,\"name\":\"Lana Wachowski\"}]," +
+                    "\"meta\":[{\"id\":6,\"type\":\"node\",\"deleted\":false}]}]}]," +
+                "\"errors\":[]}";
+        final List<TableRecord> records = CypherUtils.parse(neo4jResponse);
+        final List<SolutionMapping> solMaps = new Record2SolutionMappingTranslatorImpl()
+                .translateRecords(records, conf, translation, generator);
+        final BindingBuilder builder = Binding.builder();
+        builder.add(Var.alloc("s"), conf.mapNode(node6));
+        final SolutionMapping m1 = new SolutionMappingImpl(builder.build());
+        assertEquals(solMaps.size(), 1);
+        assertEquals(solMaps.get(0), m1);
```

Comment:
```suggestion
        assertEquals(1, solMaps.size());
        assertEquals(m1, solMaps.get(0));
```
The first argument is always the expected value and the second argument is the value to be tested!